src/prepare.c

```diff
@@ -256,6 +240,10 @@ while(0)
 	
 	if (candidate->default_properties_count) {
 		int i;
+		
+		if(prepared->default_properties_table != NULL) {
```

Comment:
zend_class_entry of anonymous classes already exists per interpreter instance at thread creation, like internal classes. But they are incomplete, especially the default_properties_table and default_static_members_table. I guess the anon class zend_class_entry gets "constructed" at compile time. But final property and static member info is not known at compile time, first at runtime.